src/expr/src/id.rs

```diff
@@ -47,6 +47,13 @@ impl LocalId {
     pub fn new(v: u64) -> LocalId {
         LocalId(v)
     }
+    /// Read the contained identifier.
+    ///
+    /// The contained value is not semantically meaningful, and this method
+    /// exists mainly to assist others in finding distinct identifiers.
+    pub fn id(&self) -> u64 {
+        self.0
+    }
```

Comment:
@benesch is this antithetical to `LocalId`? It seemed to be the most appropriate way to find a new identifier not yet used in an expression (to wrap an existing `RelationExpr` in a new `Let`). The other option is to rewrite all identifiers in the inner expression; perhaps that results in more idiomatic identifiers anyhow, I'm not sure.

Reply:
Nope! Could also just make field `0` public if you prefer. `LocalId` (to me at least) is just about making functions that take a `LocalId` parameter more self-documenting than `some_param: u64`.

Reply:
Update: the errors were due to `InlineLet` barfing because identifiers overlapped, not shadowed just overlapped. I'll probably rewrite them all with `UpdateLet` which should handle this.

Reply:
This was removed as part of a more systematic way of ensuring that all identifiers are unique (essentially, re-run `UpdateLet` logic as part of stitching together the views).